react/src/lib/Sidebar/index.js

```diff
@@ -48,14 +49,17 @@ class Sidebar extends React.Component {
 
   render() {
     const {
+      buttonProps,
       children,
       className,
       expandable,
```

Comment:
I think that we should use `collapsable` as the prop name since we are defaulting to expanded.